Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -68,6 +69,67 @@ export default function widgetBehavior(publicAPI, model) {
     model.widgetState.deactivate();
   };
 
+  publicAPI.handleStartMouseWheel = (callData) => {
+    publicAPI.invokeStartInteractionEvent();
+    // When interacting, plane actor and lines must be re-rendered on other views
+    publicAPI.getViewWidgets().forEach((viewWidget) => {
+      viewWidget.getInteractor().requestAnimation(publicAPI);
+    });
+  };
+
+  publicAPI.handleMouseWheel = (calldata) => {
+    const direction = calldata.spinY;
+    const currentCenter = model.widgetState.getCenter();
+    const image = model.widgetState.getImage();
+    const imageSpacing = image.getSpacing();
+    const imageBounds = image.getBounds();
+    const boundingBox = vtkBoundingBox.newInstance({ bounds: imageBounds });
+
+    // Get the normal use to translate the center along
+    const normal = calldata.pokedRenderer
+      .getRenderWindow()
+      .getRenderers()[0]
+      .getActiveCamera()
+      .getDirectionOfProjection();
+
+    // Direction of the projection if the inverse of what we want
+    const translationVector = vtkMath.multiplyScalar(normal, -1);
+
+    let newCenter = [
+      currentCenter[0] + direction * translationVector[0] * imageSpacing[0],
+      currentCenter[1] + direction * translationVector[1] * imageSpacing[1],
+      currentCenter[2] + direction * translationVector[2] * imageSpacing[2],
+    ];
+
+    if (!boundingBox.containsPoint(...newCenter)) {
```

Comment:
does not work when translationVector is not an orthogonal axis.

![image](https://user-images.githubusercontent.com/219044/96562026-93587200-12c0-11eb-8def-5c2bebbcda38.png)


You should use `intersectPlane`

Reply:
Move back newCenter by the shortest distance between all bbox corners.